include/fastdds/rtps/participant/RTPSParticipant.h

```diff
@@ -23,13 +23,17 @@
 #include <cstdlib>
 #include <memory>
 
-#include <fastrtps/fastrtps_dll.h>
+#include <fastdds/rtps/builtin/discovery/endpoint/EDP.h>
 #include <fastdds/rtps/common/Guid.h>
 #include <fastdds/rtps/attributes/RTPSParticipantAttributes.h>
 #include <fastdds/rtps/builtin/data/ContentFilterProperty.hpp>
 #include <fastdds/statistics/IListeners.hpp>
+#include <fastrtps/fastrtps_dll.h>
 #include <fastrtps/qos/ReaderQos.h>
 #include <fastrtps/qos/WriterQos.h>
+#include <fastrtps/types/TypesBase.h>
+
+using eprosima::fastrtps::types::ReturnCode_t;
```

Comment:
These ReturnCodes are defined only in the DDS layer (and in fact they are defined in the DDS specification but not in the RTPS one). These return codes are not used in the RTPS layer, so please removed them from here and return only a bool as it is done in the other methods in this layer.